charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.8.2
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.43.8
+version: 5.43.9
```

Comment:
I'd change minor as it's changing backwards compatibility

Reply:
Bumped - compatibility changed in 1.22 :)